### DISCUSSION TITLE
New Sprint weekend format

### DISCUSSION THREAD

**Discussion by SmCTwelve** [theOehrly/Fast-F1#357] [Ideas]
As recently [announced](https://www.formula1.com/en/latest/article.explained-everything-you-need-to-know-about-the-2023-f1-sprint-format.583WHWKbWVVBemPKi6pJxH.html) from next race, sprint weekends are now structured as:

- FP1
- Qualifying
- Sprint Qualifying
- Sprint
- Race

It would seem that `EventFormat` and `Session` identifiers may need to account for the change, since 'SQ' would no longer be synonymous with 'S' for the current season.

**Comment by theOehrly** [comment]
So first off, the new session is called "Sprint Shootout". The abbreviation for that will be 'SS' once it is implemented (that's the abbreviation that is used internally as well by F1 in their API).

I currently assume that everything should work with the current version of FastF1 as long as you load sessions by number and not by name or abbreviation.
I. e. `fastf1.get_session(2023, 4, 3)` should work, but something like `fastf1.get_session(2023, 4, 'Sprint Shootout')` won't work.

I'll check after the session and see if I need to make any other changes. But I don't think so.

**Reply by simeonericcardo** [reply]
No, at the moment both methods aren't working. " Invalid session type 'Sprint Shootout' " error

**Reply by theOehrly** [reply]
Yes, I was wrong about that. A new version with a fix is out now.

**Reply by patadeganzo** [reply]
Both methods, "Sprint Qualifying" and "SQ," are working just fine. However, for some reason, no data is being loaded while using either method.

**Reply by simeonericcardo** [reply]
No, again. Not working. Sprint Race data (void obviously) are loaded, not Sprint Qualifying

**Reply by theOehrly** [reply]
The session is called "Sprint Shootout" or "SS" in short.

Sprint Qualifying was the name of the Sprint Race in 2021 when it was used to form the starting grid for the actual race. Internally, FastF1 treats "Sprint" and "Sprint Qualifying" the same. Thank F1 for the intuitive naming.

**Reply by formulatimer** [reply]
Hi,

I have read the documentation of the session identifiers and I have tried using "SS", "Sprint Shootout" and "3" to load the Sprint Shootout session but it always return me:

\fastf1\events.py:766 in get_session_name
    raise ValueError(f"Invalid session type '{identifier}'")

ValueError: Invalid session type 'Sprint Shootout'

When I load other session of Baku, for example "Sprint", it works
I hope you can help me

**Reply by theOehrly** [reply]
@manpean please ensure that you are using the latest version of FastF1 and update if necessary

**Reply by formulatimer** [reply]
I updated the version and now it works!!! Thank you
